README.md

```diff
@@ -2,6 +2,10 @@
 # Cosmos DB Support for VS Code (Preview)
 [![Build Status](https://travis-ci.org/Microsoft/vscode-cosmosdb.svg?branch=master)](https://travis-ci.org/Microsoft/vscode-cosmosdb) [![Release Status](https://img.shields.io/github/tag/Microsoft/vscode-cosmosdb.svg?label=prerelease&colorB=0E7FC0)](https://github.com/Microsoft/vscode-cosmosdb/releases)
 
+Browse and query your MongoDB databases both locally and in the cloud using [_scrapbooks_](#mongo-scrapbooks) with rich Intellisense. Connect to Azure and manage your Cosmos DB databases with support for MongoDB, Graph (Gremlin), and DocumentDB.

```

Comment:
Would it be better to separate out these two sentences into two bullet items? With the current wording, it's easy to skim by reading just the first sentence and get the impression it's only about mongo.

Reply:
I really want the pitch to be prose. I can try to combine the sentences..

> Browse and query your MongoDB databases both locally and in the cloud using [_scrapbooks_](#mongo-scrapbooks) with rich Intellisense then connect to Azure to manage your Cosmos DB databases with support for MongoDB, Graph (Gremlin), and DocumentDB (SQL).

Reply:
ok